doc/DeveloperGuide.md

```diff
@@ -88,6 +88,36 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**Preconditions**
+
+1. Addressbook should have at least one user with tags
+
+**Guarantees**
+
+1. All users with specified tag to be renamed will be renamed
+
+**MSS**
+
+1. User enters rename tag command
+2. User specifies which tag to be renamed and what it should be renamed into
+3. Addressbook confirms tags have been renamed.
```

Comment:
AddressBook should wait for confirmation from user as stated in the LO description, so perhaps that step should be added as well.